### PR TITLE
Hero animation added

### PR DESCRIPTION
**Pull request by juanpablo-pereira**
### Animation of the transition from Movies to Movie Details with Hero

movies-animation

### CODE REVIEW

File: lib/src/widgets/movie_poster_detail.dart [hunk 1]
```diff
@@ -19,11 +19,14 @@ class MoviePosterDetail extends StatelessWidget {
...
+          child: Hero(
+            tag: '$imagePath',
+            child: Image.network(
+              '${MovieStrings.gridBuilderImageUrlStart + imagePath}',
+              fit: BoxFit.fitWidth,
+              height: MovieDimensions.detailsImageHeight,
+              width: MovieDimensions.detailsImageWidth,
+            ),
           ),
         ),
```
Comment (FranciscoGaraventa) [inline lib/src/widgets/movie_poster_detail.dart]: You are using the same structure here and in movies_grid_builder, can you abstract this Hero Widget into a new Widget, You can use the commons parameters that you use in both codes like tag, image network route, etc.
And for fit, height, width you can pass it as parameter the value that you want to use to.

Comment (juanpablo-pereira) [reply lib/src/widgets/movie_poster_detail.dart]: Done!

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**